tracing/src/collect.rs

```diff
@@ -5,16 +5,8 @@ pub use tracing_core::collect::*;
 #[cfg_attr(docsrs, doc(cfg(feature = "std")))]
 pub use tracing_core::dispatch::DefaultGuard;
 
-/// Sets this collector as the default for the duration of a closure.
-///
-/// The default collector is used when creating a new [`Span`] or
-/// [`Event`], _if no span is currently executing_. If a span is currently
-/// executing, new spans or events are dispatched to the collector that
-/// tagged that span, instead.
-///
-/// [`Span`]: super::span::Span
-/// [`Collect`]: super::collect::Collect
-/// [`Event`]: tracing_core::Event
+/// Sets this collector as the default for the current thread for the duration
+/// of a closure.
```

Comment:
let's maybe keep this part?
```suggestion
/// of a closure.
///
/// The default collector is used when creating a new [`Span`] or
/// [`Event`].
///
/// [`Span`]: super::span::Span
/// [`Event`]: tracing_core::Event
```